apps/ledger-live-mobile/src/reducers/settings.ts

```diff
@@ -473,6 +474,20 @@ const handlers: ReducerMap<SettingsState, SettingsPayload> = {
     },
   }),
 
+  [SettingsActionTypes.LAST_SEEN_DEVICE_LANGUAGE_ID]: (state, action) => {
+    if (!state.lastSeenDevice) return state; // Nb can't make you happy TS.
```

Comment:
I think it's normal to have this check on `lastSeenDevice`, because the action's `payload` is only a `Partial<DeviceInfo>` -> so if `lastSeenDevice` does not exist, we would have a `deviceInfo: Partial<DeviceInfo>` and not `deviceInfo: DeviceInfo` ;)

Reply:
So it's not that I didn't want to make it happy, I just couldn't because of the nature of our relationship. Got it.

Reply:
```suggestion
    if (!state.lastSeenDevice) return state;
```